docs/content/en/docs/reference/clusterspec/cni.md

```diff
@@ -0,0 +1,173 @@
+    cniConfig:
+      kindnetd: {}
+```
+
+> NOTE: EKS Anywhere allows specifying only 1 plugin for a cluster and does not allow switching the plugins
+after the cluster is created.
+
+### Configuration options for Cilium plugin
+
+Cilium accepts policy enforcement modes from the users to determine the allowed traffic between pods.
+The allowed values for this mode are: `default`, `always` and `never`.
+Please refer the official [Cilium documentation](https://docs.cilium.io/en/stable/policy/intro/) for more details on how each mode affects
+the communication within the cluster and choose a mode accordingly.
+You can choose to not set this field so that cilium will be launched with the `default` mode.
+Starting release 0.8, Cilium's policy enforcement mode can be set through the cluster spec
+as follows:
+
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster-name
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 192.168.0.0/16
+    services:
+      cidrBlocks:
+      - 10.96.0.0/12
+    cniConfig:
+      cilium: 
+        policyEnforcementMode: "always"
+```
+
+Please note that if the `always` mode is selected, all communication between pods is blocked unless
+NetworkPolicy objects allowing communication are created.
+In order to ensure that the cluster gets created successfully, EKS Anywhere will create the required
+NetworkPolicy objects for all its core components. But it is up to the user to create the NetworkPolicy
+objects needed for the user workloads once the cluster is created.
+
+#### Network policies created by EKS Anywhere for "always" mode
+
+As mentioned above, if Cilium is configured with `policyEnforcementMode` set to `always`,
+EKS Anywhere creates NetworkPolicy objects to enable communication between
+its core components. These policies are created based on the type of cluster as follows:
+
+1. For self-managed/management cluster, EKS Anywhere will create NetworkPolicy resources in the following namespaces allowing all ingress/egress traffic by default:
+- kube-system
+- eksa-system
+- All core Cluster API namespaces:
+    + capi-system
+    + capi-kubeadm-bootstrap-system
+    + capi-kubeadm-control-plane-system
+    + etcdadm-bootstrap-provider-system
+    + etcdadm-controller-system
+    + cert-manager
+- Infrastruture provider's namespace (for instance, capd-system OR capv-system)
+- If Gitops is enabled, then the gitops namespace (flux-system by default)
+
+This is the NetworkPolicy that will be created in these namespaces for the self-managed cluster:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-all-ingress-egress
+  namespace: test
+spec:
+  podSelector: {}
+  ingress:
+  - {}
+  egress:
+  - {}
+  policyTypes:
+  - Ingress
+  - Egress
+```
+
+2. For a workload cluster managed by another EKS Anywhere cluster, EKS Anywhere will create NetworkPolicy resource only in the following namespace by default:
```

Comment:
Likewise, I'd suggest indenting all the lines that goes with step "2." three spaces. That would have the added value of letting you number the step as "1." and still have it number properly. This is a good trick for long procedures, but not a big deal here.